weather-widget/weather.lua

```diff
@@ -450,7 +450,11 @@ local function worker(args)
         end)))
 
     -- watch('cat /home/pmakhov/.config/awesome/awesome-wm-widgets/weather-widget/weather.json', 5, update_widget, weather_widget)
-    watch(string.format(GET_FORECAST_CMD, owm_one_cal_api), 5, update_widget, weather_widget)
+    watch(
+        string.format(GET_FORECAST_CMD, owm_one_cal_api),
+        120,  -- API limit is 1k req/day; day has 1440 min; every 2 min is good
+        update_widget, weather_widget,
+    )
```

Comment:
@streetturtle Oops! it looks like the lua parser does not like this and crashes...

Reply:
@streetturtle hotfix: https://github.com/streetturtle/awesome-wm-widgets/pull/171